lib/data/providers/router_provider.dart

```diff
@@ -0,0 +1,31 @@
+import 'package:go_router/go_router.dart';
+import 'package:riverpod_annotation/riverpod_annotation.dart';
+
+import '../../constants/routes.dart';
+import '../../presentation/feature/detail/detail_page.dart';
+import '../../presentation/feature/home/home_page.dart';
```

Comment:
Go and look in the file `analysis_options.yaml` line 108 : `- prefer_relative_imports`.

If you try and make a Pull Request without using relative imports, it will fail.
Also Android Studio will warn you about this before committing, if you turn on that check...
![image](https://user-images.githubusercontent.com/11836011/202342155-c0bcc688-be69-4bda-8c01-3b8743714d5b.png)
So it is not necessary to mention in README, since anyone can read/modify the Lint file.